save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FiltersRow.kt

```diff
@@ -30,20 +30,30 @@ class SelectOption {
  * A row of filter selectors for table with `TestExecutionDto`s. Currently filters are "status" and "test suite".
  *
  * @param initialValueStatus initial value of `TestResultStatus`
+ * @param initialValueTestName initial value of `test name`
  * @param initialValueTestSuite initial value of `test suite`
+ * @param initialValueTag initial value of `tag`
  * @param onChangeStatus handler for selected value change
- * @param onChangeTestSuite handler for input value
+ * @param onChangeTestName handler for input value test name
+ * @param onChangeTestSuite handler for input value test suite
+ * @param onChangeTag handler for input value tag
  * @return a function component
  */
-@Suppress("TOO_LONG_FUNCTION", "LongMethod")
+@Suppress("TOO_LONG_FUNCTION", "TOO_MANY_PARAMETERS")
 fun testExecutionFiltersRow(
-    initialValueStatus: String,
-    initialValueTestSuite: String,
+    initialValueStatus: String = "",
+    initialValueTestName: String = "",
+    initialValueTestSuite: String = "",
+    initialValueTag: String = "",
     onChangeStatus: (String) -> Unit,
```

Comment:
Not sure it is a good idea to pass callbacks as arguments. I would recommend you to pass it through `Props`